src/algorithm/SORT-OMPTarget.cpp

```diff
@@ -0,0 +1,56 @@
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2017-20, Lawrence Livermore National Security, LLC
+// and RAJA Performance Suite project contributors.
+// See the RAJAPerf/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "SORT.hpp"
+
+#include "RAJA/RAJA.hpp"
+
+#if defined(RAJA_ENABLE_TARGET_OPENMP)
+
+#include "common/OpenMPTargetDataUtils.hpp"
+
+#include <iostream>
+
+namespace rajaperf
+{
+namespace algorithm
+{
+
+  //
+  // Define threads per team for target execution
+  //
+  const size_t threads_per_team = 256;
+
+#define SORT_DATA_SETUP_OMP_TARGET \
+  int hid = omp_get_initial_device(); \
+  int did = omp_get_default_device(); \
+\
+  allocAndInitOpenMPDeviceData(x, m_x, iend*run_reps, did, hid);
+
+#define SORT_DATA_TEARDOWN_OMP_TARGET \
+  getOpenMPDeviceData(m_x, x, iend*run_reps, hid, did); \
+  deallocOpenMPDeviceData(x, did);
+
+
+void SORT::runOpenMPTargetVariant(VariantID vid)
```

Comment:
Is this a placeholder for an eventual implementation? If not, I think it would be better to omit this file and put the message output in the header file -- less code + more obvious?

Reply:
Good point, this is another weakness of the openmp target backend. I'll move the definition into the header for brevity.

Reply:
moved